daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/SeparatedSequenceChildParseResultHelper.scala

```diff
@@ -279,7 +279,7 @@ class PositionalTrailingGroupSeparatedSequenceChildParseResultHelper(
     if (isModelGroupRepPossiblyZeroLength &&
       !isModelGroupRepNonZeroLength) {
       pstate.setSuccess()
-      ParseAttemptStatus.MissingSeparator
+      ParseAttemptStatus.AbsentRep
```

Comment:
The JIRA ticket has a small test that reproduces this problem. That test needs to be part of this change set.

Reply:
We need to run HL7 regression tests for this, as well as EDIFACT and other DFDL schemas that exercise this aspect of the code more thoroughly. 

I'm going to need the test, breakpoint this line, and walk backwards to where this is used to figure this out.

Reply:
I have added the test I used to vet this change under the UserSubmitted tests as DFDL_2399.

Reply:
I've traced this in the code. I'm convinced AbsentRep is the right return value here. 

I think this code does need future work to clean up some confusion. There are code paths being walked here that when written clearly had elements in mind, but in this test case, the schema component in consideration is a sequence group, so that suggests a strong possibility cases are missing. 

So long as a full regression on all known DFDL schemas works, I'm happy with this.